client/components/tinymce/plugins/simple-payments/dialog/form.jsx

```diff
@@ -19,121 +23,128 @@ import CompactFormToggle from 'components/forms/form-toggle/compact';
 import FormInputValidation from 'components/forms/form-input-validation';
 import UploadImage from 'blocks/upload-image';
 
-class ProductForm extends Component {
-	handleFieldChange = ( { currentTarget: { name, value } } ) => {
-		this.props.onFieldChange( name, value );
-	};
+const REDUX_FORM_NAME = 'simplePaymentsForm';
+
+// Export some selectors that are needed by the code that submits the form
+export const getProductFormValues = state => getFormValues( REDUX_FORM_NAME )( state );
+export const isProductFormValid = state => isValid( REDUX_FORM_NAME )( state );
+export const isProductFormDirty = state => isDirty( REDUX_FORM_NAME )( state );
+
+// Validation function for the form
+const validate = ( values, props ) => {
+	// The translate function was passed as a prop to the `reduxForm()` wrapped component
+	const { translate } = props;
+	const errors = {};
+
+	if ( ! values.title ) {
+		errors.title = translate( 'Product name can not be empty.' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'Product name cannot be empty.' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).